src/widgets/block.rs

```diff
@@ -191,24 +299,30 @@ impl<'a> Block<'a> {
         self
     }
 
-    /// Applies the style to all titles. If a title already has a style, it will add on top of it.
+    /// Applies the style to all titles.
+    ///
+    /// If a [`Title`] already has a style, the title's style will add on top of this one.
     pub const fn title_style(mut self, style: Style) -> Block<'a> {
         self.titles_style = style;
         self
     }
 
-    /// Aligns all elements that don't have an alignment
+    /// Aligns all [titles](Title) that don't have an [`Alignment`].
+    ///
+    /// This effectively acts as a default for titles that don't explicitly set an [`Alignment`].
```

Comment:
This can be stronger:

/// Sets the default alignment of titles...

Similar note as above about how to write about default / override with consistent language, though this one is less important as it's a pure replacement. It still would be handy to end up with the same language meaning the same thing across the library.